server/odc-service/src/main/java/com/oceanbase/odc/service/iam/auth/OrganizationAuthenticationInterceptor.java

```diff
@@ -55,6 +55,7 @@ public class OrganizationAuthenticationInterceptor implements HandlerInterceptor
             "/api/v2/iam/users/me/organizations",
             "/api/v2/iam/login",
             "/api/v2/flow/flowInstances/*/tasks/download",
+            "/api/v2/flow/flowInstances/*/tasks/log/download",
```

Comment:
why should we add a new api?

Reply:
Because I have not provide a API for the fron-end to download  full log file, and the existing APIs are not easily reusable. This issue (cannot download full log) exists since long time before.